pygmt/helpers/decorators.py

```diff
@@ -393,14 +393,14 @@ def fmt_docstring(module_func):
 
     The following are places for common parameter descriptions:
 
-    * ``{R}``: region (bounding box as west, east, south, north)
-    * ``{J}``: projection (coordinate system to use)
-    * ``{B}``: frame (map frame and axes parameters)
-    * ``{U}``: timestamp (insert time stamp logo)
-    * ``{CPT}``: cmap (the color palette table)
-    * ``{G}``: color
-    * ``{W}``: pen
-    * ``{n}``: interpolation
+    * ``{region}``: region (bounding box as west, east, south, north)
+    * ``{projection}``: projection (coordinate system to use)
+    * ``{frame}``: frame (map frame and axes parameters)
+    * ``{timestamp}``: timestamp (insert time stamp logo)
+    * ``{cmap}``: cmap (the color palette table)
+    * ``{color}``: color
+    * ``{pen}``: pen
+    * ``{interpolation}``: interpolation
```

Comment:
I think we can remove this list because the list is incomplete and there is no need to maintain it.

Reply:
Done in 9dccaa40f5035453cbf64efd3349aa7c1d76f7ca.